seatunnel-api/src/main/java/org/apache/seatunnel/api/table/type/SeaTunnelRow.java

```diff
@@ -237,7 +237,11 @@ private int getBytesForArray(Object v, SeaTunnelDataType<?> dataType) {
                 return getArrayMapNotNullSize(v);
             case NULL:
             default:
-                return 0;
+                int total = 0;
+                for (Object elem : (Object[]) v) {
+                    total += getBytesForValue(elem, dataType);
+                }
+                return total;
```

Comment:
why put it into default case. We should put it into the real datatype. For example, case ARRAY.

Reply:
This part we should handle `case ARRAY`